### PR TITLE
Milestone 1: Query and update animals table

### PR DESCRIPTION
**Pull request by mohisa302**
### Query and update the animals' table
In these screenshots I:

-  #### Inserted new data and added a new column:

<hr style="border:0.2px solid gray">

-  #### Updated by setting the species column to digimon and pokemon:

<hr style="border:0.2px solid gray">

-  #### Deleted all records:

<hr style="border:0.2px solid gray">

-  #### Updated all animals' weights that are negative to be their weight multiplied by -1:

<hr style="border:0.2px solid gray">

- #### Answered the below questions:
- How many animals are there?

- How many animals have never tried to escape?

- What is the average weight of animals?

- Who escapes the most, neutered or not neutered animals?

- What is the minimum and maximum weight of each type of animal?

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
-('Pikachu', 'Jan 7, 2021', 15.04, 'false', 1),
-('Devimon', 'May 12, 2017', 11, 'true', 5);
\ No newline at end of file
+  ('Gabumon', 'Nov 15, 2018', 8, 'true', 2),
+  ('Agumon', 'Feb 3, 2020', 10.23,'true', 0 ),
+  ('Pikachu', 'Jan 7, 2021', 15.04, 'false', 1),
+  ('Devimon', 'May 12, 2017', 11, 'true', 5);
+
+ ALTER TABLE animals
+  ADD species varchar(100);
```
Comment (muneebulrehman) [inline data.sql]: Any modification of tables should be done in the schema.sql file. Kindly move this piece of code to the schema.sql file.

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,54 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';  
+ROLLBACK;
+
+BEGIN;
+  UPDATE animals
+  SET species = 'digimon'
+  WHERE name LIKE '%mon%';
```
Comment (muneebulrehman) [inline queries.sql]: `%mon%` will match any string that contains the letters "mon" anywhere in the text. But we need to check for the name only ending in "mon". Kindly use `%mon` instead.

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,54 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
...
+  FROM animals
+  GROUP BY neutered;
+
+SELECT MAX(weight_kg), MIN(weight_kg), species
+  FROM animals
+  GROUP BY species;
+
+SELECT AVG(escape_attempts), species
+  FROM animals
+  WHERE date_of_birth between '1990-01-01' AND '2000-01-01'
```
Comment (muneebulrehman) [inline queries.sql]: We need to find the average number of escape attempts per animal type of those born between 1990 and 2000. So the date range should be from '1990-01-01' and  '2000-12-31'. Kindy update it accordingly.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @mohisa302 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (mohisa302) [reply queries.sql]: @muneebulrehman  Thanks for the review, I will apply them.

**Review by gemmen29 — APPROVED** [review]
Hi @mohisa302, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**